test-workloads/src/bin/guestvm.rs

```diff
@@ -169,6 +171,8 @@ pub fn test_vector() {
         }
     }
 
+    test_assert!(!should_panic, "writing/reading vector registers");
+
```

Comment:
This and the cert stuff below seems like good candidates for `test_runtest!`.

Reply:
These seem a bit to complicated to drop into a single block.

Reply:
Seems pretty straightforward to me. Just make `test_vector` / `test_attestation` return `Pass` / `Fail` instead of panicing on error and then: `test_runtest!("...", || test_foo())`.

Reply:
fixed